masterlist.yaml

```
@@ -7540,18 +7540,33 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/1772/' ]
     tag: [ Relev ]
 
+  - name: '(Skyrim Revamped|SR).*\.esp'
```

It might catch too many other mods, so I'd add the `-` as well.

```suggestion
  - name: '(Skyrim Revamped|SR - ).*\.esp'
```